lib/matplotlib/tests/test_patches.py

```diff
@@ -825,8 +825,9 @@ def test_annulus():
     ax.set_aspect('equal')
 
 
+@pytest.mark.parametrize('mode', ('a', 'b'))
```

Comment:
It seems like `a` is (setting by) `axis` and `b` is `radius`.

Reply:
yeah, that is what it is doing.   I went with the most obvious to me solution, making two figures with the same name is a better solution.

Reply:
On third thought, I left it paramaterized, but gave better names (and re-ordered the code a bit to only have one if/elif).